deploy/ansible/roles/metrics/files/prometheus.yml

```diff
@@ -0,0 +1,14 @@
+scrape_configs:
+    - job_name: analysis_server
+      scrape_interval: 5s
+      static_configs:
+      - targets:
+        - analysisentry-01.devnet.shimmer.iota.cafe:9312
+        - bootstrap-01.devnet.shimmer.iota.cafe:9312
+        - vanilla-01.devnet.shimmer.iota.cafe:9312
+        - drng-01.devnet.shimmer.iota.cafe:9312
+        - drng-02.devnet.shimmer.iota.cafe:9312
+        - drng-03.devnet.shimmer.iota.cafe:9312
+        - drng-04.devnet.shimmer.iota.cafe:9312
+        - drng-05.devnet.shimmer.iota.cafe:9312
+        - faucet-01.devnet.shimmer.iota.cafe:9312
```

Comment:
Are we sure having the name of these machines under the `devnet` domain is not going to create confusion with the public DevNet?

Reply:
Yeah, I think we should settle on distinct naming for all those environments:
1. Production environment (which is deployed manually with a new release) - **DevNet**
2. Development environment (which is deployed automatically from "develop" branch) - **TestNet** ???
@karimodm @jonastheis @Wollac @luca-moser

Reply:
I would keep `devnet` for the production environment and maybe something like `pre-devnet` or `test-devnet` for the develop branch. 
We use `testnet` for the internal testnet

Reply:
I vote for `pre-devnet` then

Reply:
Lets go with `pre-devnet`